src/components/pages/collectRecordFormPages/FishBeltForm/FishBeltTransectInputs.js

```diff
@@ -69,7 +69,8 @@ const FishBeltTransectInputs = ({
   const currentOptions = [...getOptions(currents.data), { label: 'not reported', value: '' }]
   const tideOptions = [...getOptions(tides.data), { label: 'not reported', value: '' }]
   const fishbelt_transect = validationsApiData?.fishbelt_transect
-  const hasFishBeltObservations = !!observationsState.length
+  // account for empty starter row
+  const hasFishBeltObservations = observationsState.length > 1
```

Comment:
recent update to fix broken tests after merging dev - we want to include the first empty row

Reply:
`!!observationsState.length` will break the tests - unless we want to go through and change them. I'm good with keeping `isNewRecord` however

Reply:
`!!observationsState.length` will make the tests fail - unless we want to update these

Reply:
`isNewRecord && observationsState.length > 1 ` ?

Reply:
hmm, this one is WEIRD. ITs do do with M804 if that helps. Im thinking about it right now

https://github.com/data-mermaid/mermaid-webapp/pull/763

I think we need to flag it as an impossible UX situation. This ticket and M804 dont work together.

Reply:
summary, the tests were failing because they caught a bug! Party!

Reply:
```
  const hasFishBeltObservations =
    !!observationsState.length > 0 &&
    Object.prototype.hasOwnProperty.call(observationsState[0], 'fish_attribute')
```